apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -120,16 +120,17 @@ export default class CoreLibrary {
     // For the calculations below, keep in mind that x and y are located at the horizontal center and the top of the sprite, respectively.
     // In other words, x and y indicate the default position of the bubble's triangular tip.
     y = Math.min(y, APP_HEIGHT);
+    let originalX = x;
```

Comment:
nit: prefer `const` over `let`

Reply:
Also I don't think `originalX` is the clearest variable name here. It implies that it's the initial value of `x`, but that's already clear from the code; ideally the variable name could indicate why we need that value. Maybe `spriteX` or something?